### DISCUSSION TITLE
Routine load writes very slowly or even does not write at all?

### DISCUSSION THREAD

**Discussion by jaredhu** [apache/doris#37080] [A - General / Q&A]
Does the lack of be's jvm cause the routine load task to run slowly? I have dozens of routine load tasks, and the jvm is the default 1g. I found that the writing is very slow or even does not write. I started to adjust the routine_load_thread_pool_size parameter of be and changed it directly to 100, but there was no change. I adjusted the Xmx parameter to 10G, and it can write quickly. Is there a detection mechanism in the code?

**Comment by sollhui** [comment]
Routine load speed is not related to JVM configuration. I think the reason for the speed change is not the JVM, it may be influenced by other variables, but it is difficult to determine where this diff is based on existing descriptions. May the reason why writing is very slow or even does not write is routine load meet some error when load.
